tools/nl/embeddings/README.md

```diff
@@ -38,8 +38,14 @@ This is a common sheet across the different index sizes.
    [`data/curated_input/`](data/curated_input/). Note down the embeddings
    file version printed at the end of the run.
 
+   To use the base default sentence_transformer model:
    ```bash
-   ./run.sh (small | medium)
+   ./run.sh -b (small | medium)
+   ```
+
+   To use the finetuned model in PROD:
+   ```bash
+   ./run.sh -f (small | medium)
```

Comment:
There is only medium fine-tuned model right?

Reply:
Yes, for now. Updated this text to remove small. But i don't think we should default to medium (in case that was your point?). I'll keep it an arg that can be used to work with other sizes (we could have a large at some point etc)